### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by Liliya-V**
Portfolio page needs to be reviewed.

### CODE REVIEW

File: aboutme.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
         <div class="hometop aboutme-hometop">
             <nav>
                 <img class="mypicture aboutme-mypicture" src="images/IMG_9169.jpg" alt="Liliya">
-                <a href="#">Contacts</a>
-                <a href="#">Portfolio</a>
```
Comment (snack-attack) [inline aboutme.html:16]: can you link your portfolio button to the portfolio.html page?

**Review by snack-attack — COMMENTED** [review]
Everything looks really good Liliya! Very clean code, very little except making sure your portfolio site is linked in your homepage :)

File: contacts.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                <h4>
+                    <p>
+                       And even any time of Western Russia day ;)
+                    </p>
+                    <p>
+                        T. <span>+44 756 410 2835</span>
+                    </p>
+                    <p>
+                        E-mail: <span>liliiavoevodina@gmail.com</span>
+
```
Comment (sjr-mayden) [inline contacts.html]: Unnecessary extra line

File: contacts.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                    </p>
+                    <p>You could also reach me on my social media, just direct message me</p>
+                    <p>and we could meet for a coffee and discuss your project.</p>
+
```
Comment (sjr-mayden) [inline contacts.html:36]: Unnecessary extra line

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Pull request merged**